tests/fetch-tests.nix

```diff
@@ -57,7 +57,10 @@ let
             inherit pkgs lib helpers;
             config = { };
           };
-          cmp-sources = import ../plugins/completion/cmp/sources/sources.nix;
+          cmp-sources = import ../plugins/completion/cmp/sources/sources.nix {
+            inherit pkgs lib helpers;
+            config = { };
```

Comment:
Why do we need those ?
It is not obvious from the content of `sources/sources.nix`.

Reply:
The idea was that entries within `sources.nix` would be able to declare (e.g.) `settingsOptions` and `extraConfig` inline, so would need access to `pkgs`, `lib`, & `helpers`.

The more I think about it the more I lean towards moving any cmp plugin that needs to have extra options _out of_ `sources.nix` and into its own file. If we take that approach, this change is unnecessary.

Reply:
Yes, got you.
Anyway, as it is not used now, let's adopt the minimal approach and start from there.